lxd/db/images.go

```diff
@@ -11,14 +11,19 @@ import (
 	"github.com/pkg/errors"
 )
 
-// ImageSourceProtocol maps image source protocol codes to human-readable
-// names.
+// ImageSourceProtocol maps image source protocol codes to human-readable names.
 var ImageSourceProtocol = map[int]string{
 	0: "lxd",
 	1: "direct",
 	2: "simplestreams",
 }
 
+// ImageType maps image types to human-readable names.
+var ImageType = map[int]string{
```

Comment:
@stgraber could this use the instance.TypeX consts, or perhaps even move this map to `instance` package as the names and IDs are the same as we use for instance type in DB.